query-service/src/integrationTest/java/org/hypertrace/core/query/service/htqueries/HTPinotQueriesTest.java

```diff
@@ -419,6 +419,19 @@ public void testAttributeExpressionQueries(
     assertEquals(expectedValue, rows.get(0).getColumn(0).getString());
   }
 
+  @Test
+  public void testNotContainsKeyQuery() throws IOException {
+    LOG.info("Not contains key query");
+    Iterator<ResultSetChunk> itr =
+        queryServiceClient.executeQuery(
+            buildQueryFromJsonFile("query4.json"), TENANT_ID_MAP, 10000);
+    List<ResultSetChunk> list = Streams.stream(itr).collect(Collectors.toList());
+    List<Row> rows = list.get(0).getRowList();
+    assertEquals(10, rows.size());
+    assertEquals(
+        "\"http.status_code\":\"200\"", rows.get(0).getColumn(0).getString().split(",")[1]);
```

Comment:
Are all the rows are having the same values? Would this always be true that the first-row first column will have this value?

Reply:
>Are all the rows are having the same values?

No

>Would this always be true that the first-row first column will have this value?

Should be as we are generating same load ?